crates/oxc_transformer/src/regexp/mod.rs

```diff
@@ -133,10 +133,11 @@ impl<'a, 'ctx> Traverse<'a> for RegExp<'a, 'ctx> {
             }
 
             let span = regexp.span;
+            let _flags_text = flags.to_string(); // TODO: How to add `&'a str` lifetime?
```

Comment:
Now I am having trouble to resolve this lifetime constraint due to my lack of Rust experience...